infrastructure/parameters/dev.parameters.json

```diff
@@ -52,6 +52,9 @@
     },
     "publicAppGATrackingId": {
       "value": "UA-139424173-2"
+    },
+    "enableSwagger": {
```

Comment:
By itself this won't apply the parameter to the Azure App Services. The `_x_.parameters.json` files contain parameters which are substituted into the ARM template during the Infrastructure build, depending on the build environment.

Reply:
Yes this needs to be added into the template.json file as a variable, then set on the appropriate resources